.circleci/config.yml

```diff
@@ -54,4 +54,60 @@ jobs:
 
       - store_artifacts:
           path: test-reports
-          destination: test-reports
\ No newline at end of file
+          destination: test-reports
+  test:
+    working_directory: ~/mnl-api
+
+    docker:
```

Comment:
any way to DRY this up since it's the same for the build job?

Reply:
great point, i'll work on figuring this out